flashinfer/gemm/gemm_base.py

```
@@ -180,6 +181,161 @@ def _fake_cutlass_segment_gemm(
+    >>> import torch.nn.functional as F
+    >>> import flashinfer
+    >>> input = torch.randn([48, 64], device="cuda", dtype=torch.bfloat16)
+    >>> weight = torch.randn([80, 64], device="cuda", dtype=torch.bfloat16).transpose(-2, -1)
+    >>> out = flashinfer.mm_bf16(input, weight)
+    >>> print(out.shape)
+    torch.Size([48, 80])
+    >>> out.dtype
+    torch.bfloat16
+    """
+    if backend != "cutlass":
+        raise ValueError(f"Unsupported backend: {backend}. Only cutlass is available.")
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)